uyuni/common-libs/uyuni-common-libs.spec

```
@@ -67,7 +67,9 @@ BuildRequires:  python-devel
 %else
 BuildRequires:  python2-devel
 %endif
+%if 0%{?rhel} && 0%{?rhel} < 8
```

```suggestion
%if 0%{?rhel} >= 8 || 0%{?suse_version}
```
We do not have this package in Debian/Ubuntu Client Tools. So we can ignore them for now.

---

Yes, this condition makes more sense as `zchunk` is required to exctract metadata by `reposync`.

---

doh! I inverted it... fixed